ensembl_dbcopy/models.py

```diff
@@ -0,0 +1,171 @@
+# This is an auto-generated Django model module.
+# You'll have to do the following manually to clean this up:
+#   * Rearrange models' order
+#   * Make sure each model has one field with primary_key=True
+#   * Make sure each ForeignKey has `on_delete` set to the desired behavior.
+#   * Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table
+# Feel free to rename the models, but don't rename db_table values or field names.
+from django.db import models
+import uuid
+
+NAME_CHOICES_GROUP = (('Vertebrates', 'Vertebrates'),
+                           ('Microbes', 'Microbes'),
+                           ('Metazoa', 'Metazoa'),
+                           ('VectorBase', 'VectorBase'),
+                           ('Plants', 'Plants'),
+                           ('WormBase', 'WormBase'),
+                           ('Compara', 'Compara'),
+                           ('Production', 'Production'))
+
+class NullTextField(models.TextField):
```

Comment:
I would push for NullTextField (and NullCharField somehow) to move onto ensembl_production app (since they are to be used accross all our apps)

Reply:
Done in latest commit